application_sandbox/overlapping_frames/triangle.vert

```diff
@@ -16,13 +16,13 @@
 #version 450
 #extension GL_ARB_separate_shader_objects : enable
 
+layout(push_constant) uniform PushConstant {
+    float time;
+} pushConstant;
+
 layout(location = 0) out vec3 fragColor;
 
-vec2 pos[3] = vec2[](
-    vec2(-0.5, 0.5),
-    vec2(0.5, 0.5),
-    vec2(0.0, -0.5)
-);
+float theta[3] = float[](2.4981, 0.6435, 4.71238898038);
```

Comment:
Same remark, could these values be rounded to something less-precise like `2.5, 0.6, 4.7` ?

Reply:
In this case, these values are fine-tuned but I don't think the fine-tuning is necessary. They are the angle values that correspond to the original triangle co-ordinates that we were using.

Since the triangle is constantly rotating, it shouldn't matter what the original orientation of the triangle is.

Reply:
OK. Can we use the less-precise values there then?

Reply:
All good, thanks.